examples/pipeline_wrappers/run_api_streaming/README.md

```diff
@@ -0,0 +1,63 @@
+# Run API Streaming Example
+
+This example shows how to stream tokens directly from `/<pipeline>/run` endpoint
+using `streaming_generator()`. Instead of waiting for the pipeline to finish and returning
+the final string, the wrapper yields streaming chunks as soon as the underlying LLM produces them.
+
+## Highlights
+
+- Implements `run_api()` to return a generator of `StreamingChunk` objects.
+- Uses the helper `streaming_generator()` from Hayhooks (no manual queue management required).
+- Demonstrates how `/run` automatically becomes a `StreamingResponse` after returning a generator.
+- Wrap the generator with `SSEStream(...)` to turn the `/run` response into SSE.
+
+```python
+from hayhooks import SSEStream, streaming_generator
+
+def run_api(self, question: str, urls: Optional[list[str]] = None):
```

Comment:
1. in the example file, we don't use SSE
2. Showing two different snippets of code in this README feels a bit confusing